lib/v2/scitation/radar.js

```diff
@@ -5,7 +5,7 @@ module.exports = {
             {
                 title: 'journal',
                 docs: 'https://docs.rsshub.app/journal.html#scitation',
-                source: ':pub.scitation.org/toc/:jrn',
+                source: 'pubs.aip.org/:pub/:jrn',
```

Comment:
The `source` field could not match any domains.
docs: https://docs.rsshub.app/joinus/new-radar.html

Reply:
I have updated `radar.js`.